src/libasr/codegen/asr_to_fortran.cpp

```diff
@@ -1232,7 +1256,15 @@ class ASRToFortranVisitor : public ASR::BaseVisitor<ASRToFortranVisitor>
         s = r;
     }
 
-    // void visit_ArrayAll(const ASR::ArrayAll_t &x) {}
+    void visit_ArrayAll(const ASR::ArrayAll_t &x) {
+        std::string r;
+        r += "all";
+        r += "(";
+        visit_expr(*x.m_mask);
+        r += s;
+        r += ")";
+        s = r;
```

Comment:
How about?

```suggestion
        visit_expr(*x.m_mask);
        s = "all(" + s + ")";
```
This seems shorter and more readable.

Reply:
Great idea; I updated your suggestion for `transpose`. But I think `all` also supports the `dim` parameter, so I updated the code accordingly.